meta/implementation.md

```diff
@@ -230,3 +230,64 @@ We have found some libraries that can be used for an implementation:
 - Julia: [Statistics.quantile](https://docs.julialang.org/en/v1/stdlib/Statistics/#Statistics.quantile!), type 7 is the default.
 - Python: [numpy](https://numpy.org/doc/stable/reference/generated/numpy.quantile.html), [pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.quantile.html), [xarray](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.quantile.html) - type 7 (called 'linear' for the interpolation parameter) is the default for all of them.
 - R: [quantile](https://stat.ethz.ch/R-manual/R-patched/library/stats/html/quantile.html) - type 7 is the default.
+
+## OGC API - Processes
+
+OGC API - Processes and OGC EO Application Packages (AP) can generally be unilized in openEO in three different ways:
+
+1. **openEO process**
+  
+   As a pre-defined process that exposes itself as a normal openEO process.
+   It is not exposed to the user that in the background an AP is invoked.
+2. **Pre-deployment through ADES**
+  
+   In addition to the openEO API, a provider can offer access to an Application Deployment and Execution Service (ADES).
```

Comment:
```suggestion
   In addition to the openEO API, a provider can offer access to an [Application Deployment and Execution Service (ADES)](https://github.com/eoepca/proc-ades).
```

Reply:
So, probably we should cite this instead since the project was moved https://github.com/ZOO-Project/ZOO-Project/blob/4e0230a50e1afb64501a0853a3e2b23c5b9add19/docs/kernel/dru.rst

Reply:
@clausmichele I've updated the PR accordingly, see https://github.com/Open-EO/openeo-processes/pull/520/commits/e7ba6866d4b3691342433ca4bf6c0189c2fb4266